cli/docs/reference/ocm_download_plugin.md

```diff
@@ -27,25 +27,22 @@ ocm download plugin [flags]
 ### Examples
 
 ```
- # Download a plugin binary with resource name 'ocm-plugin' and version 'v1.0.0'
-  ocm download plugin ghcr.io/org/component:v1 --resource-name ocm-plugin --resource-version v1.0.0 --output ./plugins/ocm-plugin
+ # Download a plugin binary with resource name 'helminput' and version 'v0.0.0-main'
+  ocm download plugin ghcr.io/open-component-model/plugins//ocm.software/plugins/helminput:0.0.0-main
```

Comment:
Is the omitted `v` on the verison intended? (Just wondering because it was included previously)

```suggestion
  ocm download plugin ghcr.io/open-component-model/plugins//ocm.software/plugins/helminput:v0.0.0-main
```

(also missing below)